test/builder/dummies.go

```diff
@@ -504,6 +513,21 @@ func DummyVirtualMachinePublishRequest(name, namespace, sourceName, itemName, cl
 	}
 }
 
+func VirtualMachineGroupPublishRequest(name, namespace string) *vmopv1.VirtualMachineGroupPublishRequest {
+	return &vmopv1.VirtualMachineGroupPublishRequest{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:       name,
+			Namespace:  namespace,
+			Finalizers: []string{"vmoperator.vmware.com/virtualmachinegrouppublishrequest"},
```

Comment:
I guess it was copied from `DummyVirtualMachinePublishRequest()` above but this is a little odd to include in the finalizer here. Is that actually needed?

Reply:
our ReconcileNormal exits right away after adding the finianzer if it is not found